### PR TITLE
adding the models

### PR DESCRIPTION
**Pull request by lucaslima18**
Estas models ainda estão em fase inicial, mas a base é esta

### CODE REVIEW

**Review by lucaslima18 — COMMENTED** [review]
as modificações foram feitas

File: apps/financial/models.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    card_type = models.CharField(max_length=2, choices=CARD_TYPE_CHOICES, default='CR')
+    limit = models.DecimalField(max_digits=20, decimal_places=2)
+
+    def __str__(self):
+        return self.name
+
+
+class FinanceIncome(models.Model):
+    FINANCE_INCOME_TYPE_CHOICES = [
+        ('DV', 'Dívida'),
```
Comment (markirius) [inline apps/financial/models.py]: use english language, in another moment, we will use a translation builltin django method

File: apps/financial/models.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
+from django.conf import settings
+from django.db import models
+from django.urls import reverse_lazy
+
+
+class Card(models.Model):
+    CARD_TYPE_CHOICES = [
+        ('CR', 'Crédito'),
```
Comment (markirius) [inline apps/financial/models.py]: use english language here too  and all another text stuff too

**Review by markirius — CHANGES_REQUESTED** [review]

**Review by markirius — CHANGES_REQUESTED** [review]
check notes on review

File: apps/financial/models.py (hunk 1 above)
Comment (lucaslima18) [reply apps/financial/models.py]: okey, i will fix that

File: apps/financial/models.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        ('DB', 'Debt'),
+        ('RD', 'Recurring debt'),
+        ('CD', 'Card debt'),
+        ('SM', 'Separate money'),
+        ('RI', 'Recurring income'),
+        ('AI', 'Additional income'),
+    ]
+    name = models.CharField(max_length=255)
+    tipo = models.CharField(max_length=3, choices=FINANCE_INCOME_TYPE_CHOICES, default='DV')
+    valor = models.DecimalField(max_digits=20, decimal_places=2)
```
Comment (markirius) [inline apps/financial/models.py]: isn't **value** instead **valor**?

File: apps/financial/models.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    
+    def __str__(self):
+        return self.name
+
+
+class Savings(models.Model):
+    name = models.CharField(max_length=255)
+    valor = models.DecimalField(max_digits=20, decimal_places=2)
```
Comment (markirius) [inline apps/financial/models.py]: same issue here.

File: apps/financial/models.py [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+    def __str__(self):
+        return self.name
+
+
+class FinanceIncome(models.Model):
+    FINANCE_INCOME_TYPE_CHOICES = [
+        ('DB', 'Debt'),
+        ('RD', 'Recurring debt'),
+        ('CD', 'Card debt'),
```
Comment (markirius) [inline apps/financial/models.py:29]: debit?

**Review by markirius — CHANGES_REQUESTED** [review]

File: apps/financial/models.py (hunk 5 above)
Comment (lucaslima18) [reply apps/financial/models.py:29]: debt == dívida

Comment (lucaslima18) [reply apps/financial/models.py:29]: debit == débito

**Review by markirius — APPROVED** [review]

**Pull request merged**